x-pack/plugins/endpoint/server/routes/resolver/utils/normalize.ts

```diff
@@ -28,34 +24,9 @@ export function extractEntityID(event: ResolverEvent) {
   return event.endpoint.process.entity_id;
 }
 
-export function extractUniqueID(entityID: string) {
-  const parsedEntityID = parseLegacyEntityID(entityID);
-  return parsedEntityID === null ? entityID : parsedEntityID.uniquePID;
-}
-
-export function getParentEntityID(events: ResolverEvent[]) {
-  if (events.length === 0) {
-    return undefined;
-  }
-  const event = events[0];
+export function extractParentEntityID(event: ResolverEvent) {
   if (isLegacyData(event)) {
-    const uniquePPID = event.endgame?.unique_ppid;
-    return uniquePPID
-      ? LEGACY_ENTITY_PREFIX + uniquePPID + LEGACY_ENTITY_DELIMITER + event.agent.id
-      : undefined;
-  }
-  return event.endpoint.process?.parent?.entity_id;
-}
-
-export function parseLegacyEntityID(
-  entityID: string
-): { endpointID: string; uniquePID: string } | null {
-  if (!entityID.startsWith(LEGACY_ENTITY_PREFIX)) {
-    return null;
-  }
-  const fields = splitN(entityID, LEGACY_ENTITY_DELIMITER, 2);
-  if (fields.length !== 3) {
-    return null;
+    return String(event.endgame.unique_ppid);
```

Comment:
@andrewstucki do we want this to be `event.endgame?.unique_ppid` because it's possible it won't be there, similar to below right?

Reply:
yeah, sorry, that's right, I should have just deleted the prefix adding stuff, I'll change it